packages/interface-mocks/src/connection-manager.ts

```diff
@@ -23,7 +24,13 @@ class MockNetwork {
     this.components.push(components)
   }
 
-  getNode (peerId: PeerId): MockNetworkComponents {
+  getNode (peerId: PeerId | Multiaddr []): MockNetworkComponents {
+    if (Array.isArray(peerId) && peerId.length > 0) {
+      peerId = peerIdFromString(peerId[0].getPeerId() ?? '')
+    } else {
+      peerId = peerId as PeerId
```

Comment:
You can use the [isPeerId](https://github.com/libp2p/js-libp2p-interfaces/blob/master/packages/interface-peer-id/src/index.ts#L37-L39) function exported from `@libp2p/interface-peer-id` - it tests the properties of the object, informs the compiler of the type and is safer than casting.